scripts/prepare_conv_pp.py

```diff
@@ -34,6 +34,14 @@
 import pandas as pd
 import yaml
 
+try:
+    import geopandas as gpd
+
+except ImportError:
+    print(
+        "No module named 'geopandas'. You need to install 'geopandas' in order to use this script."
+    )
+
```

Comment:
The actual import happens in oemof_b3.tools.geo, so the try except has to happen there.

Reply:
Done in a282a0b